addons/stock/models/stock_quant.py

```diff
@@ -127,6 +132,50 @@ def _compute_inventory_date(self):
         for quant in quants:
             quant.inventory_date = date_by_location[quant.location_id]
 
+    def _compute_last_count_date(self):
```

Comment:
could be store with a depends on `location.last_inventory_date` so we don't recompute it every time except during _apply_inventory

Reply:
The problem with the trigger on `location_id.last_inventory_date` is that it triggers all the quants linked to the location even if I change only one. What about triggering on `quantity`?